grpc-build/src/lib.rs

```diff
@@ -79,9 +79,17 @@ impl Builder {
             cmd.arg(proto);
         }
 
-        cmd.output().context(
+        eprintln!("Running {cmd:?}");
+
+        let out = cmd.output().context(
             "failed to invoke protoc (hint: https://docs.rs/prost-build/#sourcing-protoc)",
         )?;
+
+        eprintln!(
+            "---protoc stderr---\n{}\n------",
+            String::from_utf8_lossy(&out.stderr).trim()
+        );
```

Comment:
Shall we also output stdout? Just in case

Reply:
I did initially but it doesn't seem to use it, so removed for simplicity.

Reply:
Ah ok, I think it's fine then